src/pages/development/security/subresource-integrity.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Subresource Integrity
+description: Security feature that enables browsers to verify the resources they fetch.
+---
+
+# Subresource Integrity
+Subresource Integrity (SRI) is a security feature that enables browsers to verify that resources they fetch (for example, from a CDN) are delivered without unexpected manipulation. It works by allowing you to provide a cryptographic hash that a fetched resource must match.
+See [Subresource Integrity (SRI)](https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity) to learn more about SRI.
+
+## Application Support
+In order to adhere to PCI 4.0 requirements regarding verifying the integrity of scripts on payment pages, Adobe Commerce and Magento Open Source 2.4.7 and later include support for Subresource Integrity by providing integrity hashes for all javascript assets residing in the local filesystem. (This
+functionality is defined in the Magento_Csp module.)
+Please note that currently there is no SRI support for custom remote javascript resources.
```

Comment:
Probably we need to clarify here that although there is no automatic SRI coverage for custom remote scripts, merchants still can cover those by adding 'integrity' and 'crossorigin' attributes to corresponding '<script>' tags on the case by case basis.